src/qml/QFieldSettings.qml

```diff
@@ -181,6 +191,8 @@ Page {
 
                       delegate: Row {
                           width: parent ? parent.width - 16 : undefined
+                          height: !!isVisible ? undefined : 0
```

Comment:
So by default `isVisible` is assumed to be `false`? I assume `height: undefined` will automatically calculate the height, but note `isVisible` will enforce it for `false`, `undefined`, `null`, `''` and `0`. So if `isVisible` is not set (or set to `undefined`), you will get `height: 0`.

Reply:
It's set for each item in the loop, so the default value is never used. Does that work?